pkg/ccl/backupccl/backup_job.go

```diff
@@ -78,6 +79,12 @@ var BackupCheckpointInterval = settings.RegisterDurationSetting(
 
 var forceReadBackupManifest = util.ConstantWithMetamorphicTestBool("backup-read-manifest", false)
 
+var useBulkOracle = settings.RegisterBoolSetting(
+	settings.ApplicationLevel,
+	"bulkio.backup.balanced_distribution.enabled",
+	"randomize the selection of which replica backs up each range",
+	true)
```

Comment:
You want this true for this backport?  Should it be ... false at least for a bit longer?

Reply:
This is the 23.2 backport. Yes, we want it true since we’d prefer to flip it before .0 than after.